ddtrace/tracer/option.go

```diff
@@ -163,6 +166,16 @@ func newConfig(opts ...StartOption) *config {
 			WithGlobalTag(key, val)(c)
 		}
 	}
+	if v := os.Getenv("DD_HTTP_CLIENT_ERROR_STATUSES"); v != "" {
+		WithHTTPClientErrorStatuses(v)(c)
+	} else {
+		globalconfig.SetHTTPClientCodes(parseHTTPCodeRanges("400-499"))
```

Comment:
This isn't the right place to set the defaults. The defaults should be set when initialising, inside `globalconfig`. As soon as that package is loaded (`globalconfig`), the values for the HTTP codes should immediately be the right ones (default ones).